Homework 1 Phys 250 Robin James rnj13

# Problem 1 #
## Part i ##
The perimeter of a square with sides S is 4S. For the propose problem we can write the sum of N squares as:
$$ 4S + 2S+ S+ \frac{S}{2}+... \frac{4S}{2^{N-1}} = \[ \sum_{i=0}^{N-1} \frac{4S}{2^{i}} \] = 4S(\frac{1-(.5S)^N}{1-.5S})$$
As N -> $ \infty $ the perimeter goes to  $$p=\frac{4S}{1-\frac{S}{2}} $$
## Part ii ##
The area of the square with sides S is $S^2$. The sum of the area is:
$$ \[ \sum_{i=0}^{N-1} \frac{S^2}{2^{i}} \]= 2^{1-N}(2^N-1)S^2 $$
As N -> $\infty$:
$$A=2S^2$$


## Part iii ##
$$\epsilon = Real-Numerical $$
For Part i,
$$ Real-Numerical =\frac{4S}{1-\frac{S}{2}}-4S(\frac{1-(.5S)^N}{1-.5S}) $$
For part ii
$$ Real-Numerical =2S^2-2^{1-N}(2^N-1)S^2 $$



## Part iv ##
For the perimeter:
$$10^-7=\frac{4S}{1-\frac{S}{2}}-4S(\frac{1-(.5S)^N}{1-.5S}) => $$When S =1, N is equal to 27 since it must be an integer

For the area:
$$ 10^7=2S^2-2^{1-N}(2^N-1)S^2 => \frac{(-(ln(\frac{1}{s^2})-7ln(5)-8ln(2)))} {ln(2)} $$
N is equal to 25 when S =1

The two answers do not agree for $\epsilon = 10^{-7}$

# Problem 2 #
Set your coordinate "a" to be parallel to the incline of the plane. In the reference frame, the normal force and perpendicular part of the gravitational force. We only need to use Newton's law in the "a" direction
$$ F_{net{= mgsin( \theta) $$
$$ a_a=gsin(\theta) $$
We can then plug back in to a kinematic equation to get x(t).
$$ d= \frac{1}{2}gsin(\theta)t^2 $$
## Part ii ##
$$ 4= .5 g sin(\theta) => \theta = arcsin(\frac{8}{g})= 0.955 rad  $$
## Part iii ##
$$ 0= \frac{g}{4\omega^2}(e^{\omega}-e^{-\omega}-2 sin(\omega)) -4  $$
To solve this I will use scipy root finder

In [1]:
import numpy as np
import scipy.optimize as opt

def f(w):
    return (9.8/(4.*w**2.))*(np.exp(w)-np.exp(-1*w)-2.*np.sin(w))-4.

Now with the function defined, we need to find two numbers where we think the root is.Since we know $\omega$ is an angular speed in rad/sec so we can make a guess that it will be greater than 0.01 but less than pi.

In [2]:
opt.brentq(f,0.01,3.1,xtol=10e-15)

2.3612521534858644

Running using brentq, we see that $\omega = 2.3612521534858644$rad/s
## Part iv ##
Same proccess as last part.

In [3]:

def f(w):
    return (9.8/(4.*w**2.))*(np.exp(w)-np.exp(-1*w)-2.*np.sin(w))-8.


The speed must be faster then last one but still less than 2pi

In [4]:
opt.brentq(f,2.36,6,xtol=10e-15)

3.855403679362072

To slide 8 meters, $\omega=3.855403679362072$ rad/s

# Problem 3 #
We first subsite the two equations into each other.
$$x_{n+1}=2a-x_n$$
$$ \frac{f(x)}{x_n-x{n+1}}=f'(x)$$
$$ \frac{f(x)}{x_n-x{n+1}}=\frac{dy}{dx}$$
$$ \frac{f(x)}{x_n-(2a-x_n)}=\frac{dy}{dx}$$
$$ \frac{dx}{2(x_n-a)}=\frac{dy}{f(x)}$$
$$ .5ln|x-a|=ln|f(x)|$$
When then can intrate both sides
$$A\sqrt{|x-a|}=f(x) $$

Note: I cannot figure out where the sign(x-a) term comes from

Part 2 and 3 on paper because mostly scratch work but in summary:
Part 2 converges in one iteration with the root at 2.
Part 3 starts off looking like it will not converge but then changes at the fifth iteration and starts approaching 2.